frontend/js/utils/Utils.js

```diff
@@ -126,6 +126,8 @@ var Utils = {
   },
 
   userDisplayStr: function(userInfo) {
+    if(userInfo === null) return "";
```

Comment:
It happens if you try to go to a game that doesn't exist. We need a different "game not found" page for this case.